src/Php81/README.md

```diff
@@ -5,6 +5,7 @@ This component provides features added to PHP 8.1 core:
 
 - [`array_is_list`](https://php.net/array_is_list)
 - [`MYSQLI_REFRESH_REPLICA`](https://www.php.net/manual/en/mysqli.constants.php#constantmysqli-refresh-replica) constant
+- `CURLStringFile` class
```

Comment:
I would mention that this is only polyfilled for PHP 7.4+, not for older versions.